java/test/jmri/jmrit/throttle/ThrottleFramePropertyEditorTest.java

```diff
@@ -18,10 +19,12 @@ public class ThrottleFramePropertyEditorTest {
     @Test
     public void testCtor() {
         Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        // this will disable svg icons, batik randomly crashes JUnit tests
+        InstanceManager.getDefault(ThrottlesPreferences.class).setUseLargeSpeedSlider(false);
```

Comment:
Do you see this locally or with GitHub CI tests? I have restarted many CI tests on GitHub but I don't remeber that I have seen this.

Reply:
both, locally and on github, if this line is removed, NPE ar thrown, with a Batik originating stack, but never the same. Tried several things and couldn't fix it, and never got them when running the application.